ZeroWasteFriendly/Scenes/Library/LibraryController.swift

```diff
@@ -56,6 +68,13 @@ extension LibraryController: LibraryPresentable {
         postsDataSource = posts
         tableView.reloadData()
     }
+
+    func animateCarouselHeight(height: CGFloat) {
+        UIView.animate(withDuration: 0.25) {
+            self.collectionViewHeight.constant = height
```

Comment:
never never change constants of a constraints

You can always make 2 constraints, one with lower priority, and other that can be turnedON and off
and you only animate: constraint.isActive= true/false

so you don't havee to use weird constants in code